### ISSUE TITLE
[A11yInspect] Fail: 1.3.1 A in /home/runner/work/test-app/test-app/src/App.tsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#23]
## ❌ Accessibility Fail

**Rule:** 1.3.1 A
**File:** `/home/runner/work/test-app/test-app/src/App.tsx`
**Line:** 77, **Column:** 1

### Description
List element with no list items (Major)

### Context
```html
<ul>
        <div>Item 1</div>
        <div>Item 2</div>
      </ul>
```

### Selector
`html > body > div > audio > ul`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
